packages/create/src/generators/testing-karma/templates/_package.json

```
@@ -3,7 +3,9 @@
     "test": "karma start --coverage",
     "test:watch": "karma start --auto-watch=true --single-run=false",
     "test:legacy": "karma start --legacy --coverage",
-    "test:legacy:watch": "karma start --legacy --auto-watch=true --single-run=false"
+    "test:legacy:watch": "karma start --legacy --auto-watch=true --single-run=false",
+    "update-snapshots": "karma start --update-snapshots",
```

```suggestion
    "test:update-snapshots": "karma start --update-snapshots",
```

I think it's test specific so it should have the test prefix right?

---

Good point, I've adjusted it everywhere